build.gradle

```diff
@@ -50,20 +50,19 @@ dependencies {
     implementation 'software.amazon.awssdk:sqs'
     implementation 'software.amazon.awssdk:dynamodb'
 
-    // TODO: Transient dependency patches.
-    implementation 'ch.qos.logback:logback-core:1.2.13'
-    implementation 'ch.qos.logback:logback-classic:1.2.13'
-
     // TODO: Once we upgrade amazon-sqs-java-messaging-lib to v2 we can remove aws-java-sdk-core and sqs.
     implementation 'com.amazonaws:amazon-sqs-java-messaging-lib:1.1.2'
```

Comment:
Low priority if this solution works, though I think we should be using a newer version 😄!

If you look at the dependencies of this version it utilises `javax.jms` where we should really be using `jakarta.jms` - which is used in newer versions of this!

Worth noting!